From 21c01db1c6db9e9d72ab962a0276dbb00ca31a9e Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 20 Jul 2018 14:13:57 -0700
Subject: [PATCH] TRITON-493 '--tree' option to sdc-imgadm, updates-imgadm,
 joyent-imgadm to show origin hierarchy (update sdc zone to latest sdc-imgadm)

---
 package.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package.json b/package.json
index dbf2f8b..ee9ecd9 100644
--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
         "smartdc": "git+https://github.com/joyent/node-smartdc.git#4308fd9",
         "manta": "git+https://github.com/joyent/node-manta.git#b277acf",
         "moray": "git+https://github.com/joyent/node-moray.git#b84ef0e",
-        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.2.5",
+        "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.3.0",
         "bunyan": "1.3.3",
         "assert-plus": "0.1.5",
         "ldapjs": "git+https://github.com/mcavage/node-ldapjs.git#f213b3e",
-- 
2.21.0

